newbank/server/commands/Logout.java

```diff
@@ -0,0 +1,30 @@
+package newbank.server.commands;
+
+import newbank.client.ExampleClient;
+import newbank.server.NewBankClientHandler;
+import newbank.server.NewBankServer;
+import newbank.server.customers.CustomerID;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Logout implements Command {
+
+    public CommandResponse process(ArrayList<String> anArgsList) throws CommandException {
+        if (anArgsList.size() != 1) {
+
+            String myException = "Invalid arguments, anArgsList contains: ";
+            myException = myException.concat(anArgsList.toString());
+            throw new CommandException(myException);
+        } else {
+
+            String myResponse = "Logout Successful";
```

Comment:
Ideally, we shouldn't use strings like this for this kind of check for several reasons, one of which is: if we decide to change this later, then we would have to change it both here and in the `run` method of NewBankClientHandler.

A very simple solution could be to create a constants class, that has a variable `LOGOUT_RESPONSE`. We can then return `Constants.LOGOUT_RESPONSE` here and check for that in the `run` method

Reply:
Agreed, good idea, I've created the class now and added the variable. Also removed some unused imports.